api/metric/measure.go

```diff
@@ -56,46 +64,66 @@ func newMeasure(name string, valueKind ValueKind, mos ...MeasureOptionApplier) C
 	return m
 }
 
+// NewFloat64Measure creates a new measure for float64.
 func NewFloat64Measure(name string, mos ...MeasureOptionApplier) (c Float64Measure) {
 	c.CommonMetric = newMeasure(name, Float64ValueKind, mos...)
 	return
 }
 
+// NewInt64Measure creates a new measure for int64.
 func NewInt64Measure(name string, mos ...MeasureOptionApplier) (c Int64Measure) {
 	c.CommonMetric = newMeasure(name, Int64ValueKind, mos...)
 	return
 }
 
+// GetHandle creates a handle for this measure. The labels should
+// contain the keys and values specified in the measure with the
+// WithKeys option.
 func (c *Float64Measure) GetHandle(labels LabelSet) (h Float64MeasureHandle) {
 	h.Handle = c.getHandle(labels)
 	return
 }
 
+// GetHandle creates a handle for this measure. The labels should
+// contain the keys and values specified in the measure with the
+// WithKeys option.
 func (c *Int64Measure) GetHandle(labels LabelSet) (h Int64MeasureHandle) {
 	h.Handle = c.getHandle(labels)
 	return
 }
 
+// Measurement creates a Measurement object to use with batch
+// recording.
 func (c *Float64Measure) Measurement(value float64) Measurement {
 	return c.float64Measurement(value)
 }
 
+// Measurement creates a Measurement object to use with batch
+// recording.
 func (c *Int64Measure) Measurement(value int64) Measurement {
 	return c.int64Measurement(value)
 }
 
+// Record adds a new value to the list of measure's records. The
+// labels should contain the keys and values specified in the measure
+// with the WithKeys option.
 func (c *Float64Measure) Record(ctx context.Context, value float64, labels LabelSet) {
 	c.recordOne(ctx, NewFloat64MeasurementValue(value), labels)
 }
 
+// Record adds a new value to the list of measure's records. The
+// labels should contain the keys and values specified in the measure
+// with the WithKeys option.
```

Comment:
Maybe rephrase the above to "The labels should contain the keys and values for each key specified in the measure with the WithKeys option"
What is the expected behavior if it doesn't have all the keys-value pair in the label set?